docs/security/trust/trust_sandbox.md

```diff
@@ -101,8 +101,6 @@ So, you'll need an entry for both the servers in your local `/etc/hosts` file.
         RUN chmod +x /usr/bin/docker \
```

Comment:
I presume this document used `master` because this code wasn't released until 1.8.0 - so now's the time :)